web/themes/custom/novel/novel.theme

```diff
@@ -122,6 +123,9 @@ function novel_preprocess_dpl_react_app__loan_list(array &$variables): void {
 function novel_preprocess(array &$variables): void {
   $directory = $variables['directory'] ?? 'web/themes/custom/novel';
   $variables['baseIconPath'] = "/{$directory}/assets/dpl-design-system/icons";
+
+  $config = \Drupal::config('dpl_library_agency.general_settings');
+  $variables['opening_hours_url'] = $config->get('opening_hours_url') ?? GeneralSettings::OPENING_HOURS_URL;
```

Comment:
Please use another process function for this. 

Using *_preprocess() will add the variable to _all_ templates. That is probably not what you intended. Also consider moving the preprocess function to the `dpl_library_agency` module as that is the module responsible for the configuration.